sdk/appservice/arm-appservice/review/arm-appservice.api.md

```diff
@@ -1471,6 +1468,9 @@ export interface AuthPlatform {
 // @public
 export type AuthType = "Anonymous" | "UserCredentials" | "SystemIdentity" | "UserAssigned";
 
+// @public
+export type AutoGeneratedDomainNameLabelScope = "TenantReuse" | "SubscriptionReuse" | "ResourceGroupReuse" | "NoReuse";
```

Comment:
auto generated? duplication?

Reply:
this is not a duplication, it's name is AutoGeneratedDomainNameLabelScope in swagger 
![image](https://github.com/user-attachments/assets/4a1ad68a-7552-4e78-9a10-c2b717101b8a)